lib/converters/circuit-to-3d.ts

```diff
@@ -414,22 +568,23 @@ export async function convertCircuitJsonTo3D(
   // Create a default camera positioned to view the board or components
   let camera: Camera3D
 
-  if (pcbBoard) {
+  if (primaryBoard) {
```

Comment:
don't think this is the right way to center the camera- you should either center on the board or the panel, no need for this primaryBoard variable- just add pcbPanel and the associated logic

i would also try to reduce the code in this file by making it a separate function, `getBestCameraPositionFromCircuitJson(circuitJson)` that has the handling for both pcb_board and pcb_panel